### PR TITLE
Add linters and black formatting to tox

### PR DESCRIPTION
**Pull request by t-persson**
### Applicable Issues

fixes: #39 

### Description of the Change

Add code rules in the form of 'black', 'pylint' and 'pydocstyle'.
Changes made to code to follow the code rules.
Ran black on code base
Added generic documentation for pep257

### Alternate Designs

Documentation could be fixed.
We could add more tools and linters or even change them. I went with what I like.

### Benefits

More time writing code

### Possible Drawbacks

Annoyances because of 'black' formatting.

### Sign-off

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

Signed-off-by: Tobias persson <tobias.persson@axis.com>

### CODE REVIEW

**Comment by t-persson** [comment]
Black does not work with python 3.5

**Review by magnusbaeck — APPROVED** [review]
Thanks!

File: tox.ini [hunk 1]
```diff
@@ -17,3 +17,27 @@ setenv = MONGODB_HOST = localhost
          MONGODB_REPLICASET = 
 commands =
     pytest -s --log-format="%(levelname)%: %(message)s" {posargs}
+
+[testenv:black]
+deps =
+    black
+commands =
+    black --check --diff src/eiffel_graphql_api
```
Comment (magnusbaeck) [inline tox.ini:25]: Any reason we're passing just that directory? Doing so leaves out the tests.

Comment (t-persson) [reply tox.ini:25]: No reason at all. I missed that one completely.

Comment (t-persson) [reply tox.ini:25]: #44

**Pull request merged**